rskj-core/src/test/java/co/rsk/peg/federation/FederationSupportImplTest.java

```diff
@@ -2397,6 +2397,42 @@ void voteFederationChange_addFederatorPublicKeyWithLessThanMofNVotes_returnsSucc
 
         }
 
+        @Test
+        void voteFederationChange_addFederatorPublicKey_returnsExpectedPendingFederationSizeResponseCode() {
+
+            // Arrange
+
+            BtcECKey expectedBtcECKey = new BtcECKey();
+
+            Transaction tx = TransactionUtils.getTransactionFromCaller(signatureCache, FederationChangeCaller.FIRST_AUTHORIZED.getRskAddress());
+            Transaction tx2 = TransactionUtils.getTransactionFromCaller(signatureCache, FederationChangeCaller.SECOND_AUTHORIZED.getRskAddress());
+
+            ABICallSpec createFederationAbiCallSpec = new ABICallSpec(FederationChangeFunction.CREATE.getKey(), new byte[][]{});
+
+            // Voting with  m of n authorizers to create the pending federation
+            federationSupport.voteFederationChange(tx, createFederationAbiCallSpec, signatureCache, bridgeEventLogger);
+            federationSupport.voteFederationChange(tx2, createFederationAbiCallSpec, signatureCache, bridgeEventLogger);
```

Comment:
this should be part of `Act` section

Reply:
I don't think so. Because we are not testing that the federation is created here, it has been tested before already. This test is testing that federators can be added to the pending federation, and that's what's inside the `Act` section.

Reply:
you are right! thanks